packages/components/src/Radio/Radio/Radio.module.scss

```diff
@@ -0,0 +1,101 @@
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/color";
+@import "../../../styles/forms";
+@import "../../../styles/forms-variables";
+
+$radio-size: 24px;
+$icon-size: 10px;
+$icon-offset: 5px;
+$focus-ring-offset: 2px;
+
+$dt-color-radio-background-color-hover: $color-gray-100;
```

Comment:
I think background hover color for our inputs are gray-200 now

![Screenshot 2023-09-27 at 2 46 46 pm](https://github.com/cultureamp/kaizen-design-system/assets/36558508/48699f71-e526-4e36-bfdc-b33cafd0b039)

![Screenshot 2023-09-27 at 2 47 47 pm](https://github.com/cultureamp/kaizen-design-system/assets/36558508/f273f782-a13e-4d06-8f57-8de5d62fb742)

Reply:
Nice catch